Authorization/Authorization/Presentation/Login/SignInView.swift

```diff
@@ -29,6 +30,20 @@ public struct SignInView: View {
                     .resizable()
                     .edgesIgnoringSafeArea(.top)
             }.frame(maxWidth: .infinity, maxHeight: 200)
+            let config = Container.shared.resolve(ConfigProtocol.self)
```

Comment:
Doesn't this break the Preview?
It would be better to pass dependencies to SignInView init or ViewModel

Reply:
Done, now using it from the `viewModel`.